tao_compiler/mlir/disc/transforms/lhlo_legalize_roots_to_loops.cc

```diff
@@ -650,7 +650,8 @@ LogicalResult lowerWithScheduleColReduction(
 
   SmallVector<Value, 4> yield_values_for_if;
 
-  ValueRange load_index({row_index, col_index});
+  SmallVector<Value, 2> multidim_load_index({row_index, col_index});
+  ValueRange load_index(multidim_load_index);
```

Comment:
`SmallVector` could be cast `ValueRange` implicitly i think ...

Reply:
yes it is~